data_specification/memory_region.py

```diff
@@ -15,99 +15,5 @@
 
 
 class MemoryRegion(object):
-    """ Memory region storage object.
+    """ Identifies something as a Memory region
```

Comment:
Seems to me like this should be renamed to AbstractMemoryRegion if it is just going to be an empty base class for the Real and Reference classes?

Reply:
Makes sense - not sure why I didn't as that also solves the problem of having things reference the old version of this!  I will rename...

Reply:
Fixed